reactfire/auth/index.tsx

```diff
@@ -54,7 +54,7 @@ export function AuthCheck({
   fallback,
   children,
   requiredClaims
-}: AuthCheckProps): React.ReactNode {
+}: AuthCheckProps): JSX.Element {
```

Comment:
This and the change on line 36 cause a typescript compile error because they conflict with the props declared on line 47 and 48